### ISSUE TITLE
Blank screen on startup

### ISSUE THREAD

**Issue by pierrepinon** [MicrosoftEdge/WebView2Feedback#803]
Hello,

I use Webview2 example from Microsoft web site. I try to use standalone fixed webview to deploy apps to many devices.

On some devices with same W10 version, the webview doesn't start. So, standalone webview need runtime dependencies like .NET or C++ runtime ?

Best regards

**Comment by pierrepinon** [comment]
The EBWebView folder is created, so I think the runtime work fine. But my apps doesn't display website. 

I use only demo code to avoid program error

**Comment by champnic** [comment]
@pierrepinon Could you add some details about what your app is trying to do? Is it a C++ or .NET app? What version of the SDK and runtime are you using? Does you app work locally on your dev machines?

**Comment by pierrepinon** [comment]
@champnic 

I tried both C++ and .NET app from https://github.com/MicrosoftEdge/WebView2Samples. I download fixed runtime  in version 87.0.664.75 and i use 1.0.721-prerelease package in VS

On my dev machine, the programm work fine, but on some devices, the webview isn't displayed. On this devices, rigth click, doesn't display contextual menu.

On C++ app, i display MessageBox with HRESULT after each line on callback, all time message is "Operation successful" on all broken devices.

**Comment by pierrepinon** [comment]
Issue #811 is too close, but i try with local folder, not network

**Comment by goldwaving** [comment]
Having the same problem.  Everything seems to work, but the window parenting the web view is blank.  I know that the page has loaded and is working because the javascript on the page runs as it should, but the web view is not painted in the window.  Using 1.0.664.37 and latest evergreen install (87.0.664.75).

**Comment by pierrepinon** [comment]
@champnic Have you an idea about this issue ?

**Comment by champnic** [comment]
Sorry for the delay. If you are using the fixed version runtime, you will need to specify in the app where to look for the runtime binaries. Does the program work if you use the Evergreen runtime instead?

If you want to use the Fixed Version, you can specify the location with the `browserExecutablePath` parameter of `CoreWebView2Environment.CreateAsync()`: https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createasync?view=WebView2-dotnet-1.0.664.37

**Comment by noxe** [comment]
In our case we do specify the path - but the app does not work in any case when started over network

**Comment by pierrepinon** [comment]
@champnic I have already set browserExecutablePath parameter to create WebView2Controller. I have try also with Evergreen runtime and it's the same for device who the program stay blank.

All times the WebView2 Runtime start fine because, user date folder are created.

**Comment by champnic** [comment]
Thanks for the added info. If you run under Visual Studio with the debugger, is it generating any errors? If not, can you try expanding the list of errors Visual Studio catches and see if that generates anything?
![image](https://user-images.githubusercontent.com/45700886/105562342-a8350f00-5cce-11eb-95b0-53259297f5a0.png)

**Comment by pierrepinon** [comment]
I have some news about the issue. On some device, I think my company's proxy configured in Windows GPO has blocked the process. 

By error, I tried to start my program last week without network connectivity and devices display the Webview with a network error message. After this issue the Webview works all times in all conditions ! So, I'm concluding that proxy is the mistake.

I released new version with "--no-proxy-server" additional argument and it's work fine.
